libsolidity/formal/SMTEncoder.cpp

```diff
@@ -508,11 +502,7 @@ void SMTEncoder::endVisit(UnaryOperation const& _op)
 			indexOrMemberAssignment(*subExpr, newValue);
 		}
 		else
-			m_errorReporter.warning(
-				1950_error,
-				_op.location(),
-				"Assertion checker does not yet implement such increments / decrements."
-			);
+			solAssert(false, "");
```

Comment:
I see this visitor supports bitwise not, but do we have a test case for bitwise not on fixedbytes? `~bytes32(hex"1234")`

Reply:
Good point, it seems like bitwise operations for FixedBytes are actually buggy in general, since the SMTChecker treats them as numbers but doesn't shift left (if needed) before applying the operation

Reply:
Although I can't find a buggy case...

Reply:
Created https://github.com/ethereum/solidity/issues/10439 to track the bug. Will add the buggy test in this PR.